pypureclient/flasharray/client.py

```diff
@@ -5,36 +5,37 @@
 from . import PureError
 
 fa_modules_dict = {
-    '2.0': 'FA_2_0',
```

Comment:
I'm not really a big fan on this random reorder - I realize this is generated, but can we at least figure out why it's generating this mess and solve it until next release?